packages/shared/styles/components/SfNotification.scss

```diff
@@ -0,0 +1,87 @@
+@import '../variables';
+$notification__message-padding-left: 0.625rem !default;
+$notification--warning-color: #d12727 !default;
+$notification--alert-color: #0468db !default;
+$notification-padding: $spacer-big !default;
+$notification__title-font-size: $h2-font-size-desktop !default;
+.sf-notification {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    box-sizing: border-box;
+    border-radius: 15px;
+    min-height: 9rem;
+    padding: $notification-padding;
+    &__container {
+        position: relative;
+        width: 100%;
+        background: rgba(0, 0, 0, 0.8);
+        color: $c-white;
+        @media screen and (min-width: $desktop-min) {
+            display: flex;
+            flex-direction: row;
+            align-items: center;
+            min-width: 25%;
+            padding-bottom: 0;
+            border: 0;
+            text-align: center;
+            box-shadow: 0px 8px 11px rgba(0, 0, 0, 0.1);
+        }
+    }
+    &__content {
+        text-align: left;
+        font-family: $body-font-family-primary;
+        font-style: normal;
+        font-weight: 300;
+        font-size: $notification__title-font-size;
+        line-height: 24px;
+        letter-spacing: 0.02em;
+        @media (min-width: $desktop-min) {
+            font-size: $notification__title-font-size;
+        }
+    }
+    &__title {
+        font-weight: 600;
+        line-height: 1.38;
+        padding: 0;
+    }
+    &__message {
+        margin-top: 5px;
+        line-height: 1.64;
+        font-weight: 300;
+    }
+    &__icon {
+        display: none;
+        @media (min-width: $desktop-min) {
+            display: block;
+            height: 50%;
+            width: auto;
+            padding: $spacer;
+        }
+    }
+    &__close {
+        position: absolute;
+        right: 4px;
+        top: 4px;
+        @media (min-width: $desktop-min) {
+            font-size: $notification__title-font-size;
+            right: 10px;
+            top: 10px;
+        }
+    }
+    &--right {
+        text-align: right;
```

Comment:
```suggestion
```

Reply:
Should I remove this css modifier or just change css property ?

Reply:
@AdamPawlinski I can't find a design for this case. In all notification, I see text-align to right by default. Did you find text-align: center / right case